spec/controllers/projects/contributions_controller_spec.rb

```diff
@@ -110,6 +110,28 @@
     end
   end
 
+  describe 'unavailable contribution projects' do
+    before do
+      @project = create(:project, available_for_contribution: false)
+      @project_reward = create(:reward)
+      @project.rewards << @project_reward
```

Comment:
use `let` to create these vars ;)
btw, I think that instead of setting the `@project_reward` into `@project.rewards`, you can do `create(:reward, project: project)`